boa/src/class.rs

```diff
@@ -74,7 +74,9 @@ pub trait Class: NativeObject + Sized {
     /// The amount of arguments the class `constructor` takes, default is `0`.
     const LENGTH: usize = 0;
     /// The attibutes the class will be binded with, default is `writable`, `enumerable`, `configurable`.
-    const ATTRIBUTE: Attribute = Attribute::all();
+    const WRITABLE: Option<bool> = Some(true);
+    const ENUMERABLE: Option<bool> = Some(true);
+    const CONFIGURABLE: Option<bool> = Some(true);
```

Comment:
I think this should be an `Attribute`, since we want a complete data descriptor here too.

Reply:
Fixed